palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/request/GetResourcesByIdRequest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2019 Crown Copyright
```

Comment:
```suggestion
 * Copyright 2020 Crown Copyright
```
I would be inclined to update this, along with all the other GetResourcesByXXX classes, if they were moved from common.